operator/roles/forkliftcontroller/templates/api/validatingwebhookconfiguration-forklift-api.yml.j2

```diff
@@ -25,7 +25,7 @@ webhooks:
   namespaceSelector: {}
   objectSelector:
     matchExpressions:
-    - key: createdForProviderType
+    - key: createdForResourceType
```

Comment:
there is still usage of `createdForProviderType` in the admitter. did resource replace provider? should we have both? only one is enough in order to detect we should valid the entity?

Reply:
@liranr23 there are three labels we use: createdForResourceType, createdForResource and createdForProviderType. we cannot replace createdForProviderType for two reasons:
1. we already documented in 2.4 that users can specify createdForProviderType when working with CLI (without the others)
2. imagine we label a secret with createdForResourceType=providers + createdForResource=<provider name> - it's not enough since we still need to know what type of provider it is (and we can't rely on having it in the database - as a matter of face, we rely on validating the secret before the 'provider' entity is posted)

so back to your questions  - we need to keep all of them and update the documentation accordingly

Reply:
right, so if `createdForResourceType` exist, it guarantees that  `createdForProviderType` exists?

Reply:
nope, createdForProviderType exists only in secrets for providers

Reply:
ok. what i'm trying to ask is: if `createdForResourceType` cover both and we won't miss something?

Reply:
@liranr23 it doesn't cover both, we need all createdForResourceType, createdForResource and createdForProviderType to validate secrets for providers